src/util/customTransformer-faas.js

```diff
@@ -157,8 +159,6 @@ async function runOpenFaasUserTransform(
     ),
     testMode,
   );
-  stats.timing('run_time', invokeTime, tags);
```

Comment:
we have an alert on this stat. Python latency alert

Reply:
I can keep it for a bit longer but we can always migrate the alert to new stat once it is released right ?

Reply:
Pointing out as we might miss this after release.
You can remove unused `events_to_process` & `run_time` from prometheus.js file